lambdas/tests/unit/handlers/test_search_patient_details_handler.py

```diff
@@ -18,25 +19,33 @@ def patch_env_vars():
         yield env_vars
 
 
-def skip_test_lambda_handler_valid_id_returns_200(
-    valid_id_event, context, mocker, patch_env_vars
+def test_lambda_handler_valid_id_returns_200(
+    valid_id_event_with_auth_header, context, mocker, patch_env_vars
 ):
     response = Response()
     response.status_code = 200
     response._content = json.dumps(PDS_PATIENT).encode("utf-8")
 
-    mocker.patch("ssm_service.get_ssm_parameter")
+    mocker.patch.object(SSMService, "get_ssm_parameter", return_value="mock_public_key")
+    mocker.patch(
+        "jwt.decode",
+        return_value={
+            "selected_organisation": {"org_ods_code": "Y12345"},
+            "repository_role": "GP_ADMIN",
+        },
+    )
+    mocker.patch.object(SSMService, "get_ssm_parameter", return_value="1")
```

Comment:
If you need the same method to return two different return values you can use side_effect=[return_value_1, return_value_2]

Reply:
Done